install_config/topics/ocp_osp_base_image.adoc

```diff
@@ -0,0 +1,52 @@
+==== Creating a Red Hat Enterprise Linux Base Image
+
+After the user and project are created, a cloud image is required for all the
+{rhosp} instances that are to install {product-title}. For this particular reference
+environment the image used is {rhel} {rhel_version}.
+
+NOTE: {rhel} {rhel_version} includes support for overlay2 file system used in
+this reference environment for container storage purposes. If using prior
+{rhel} versions, do not use overlay2 file system.
+
+The {rhel} {rhel_version} cloud image is located at https://access.redhat.com/downloads/content/69/ver=/rhel---7/{rhel_version}/x86_64/product-software[Red Hat Enterprise Linux {rhel_version} KVM Guest Image]
+Copy the image to the director or workstation node and store the image within
+`glance`. https://docs.openstack.org/glance/latest/[Glance] image services provide the ability to discover, register
+and retrieve virtual machine (VM) images.
+
+NOTE: Depending on the _glance_ storage backend, it can be required to convert
+the image format from _qcow2_ to a different format.
+For _Ceph_ storage backends, it is https://access.redhat.com/solutions/2434691[required] the image to converted to _raw_
+image format.
+
+The following steps show the process of converting the_qcow2_ image to the
+_raw_ format for the use of a _Ceph_ storage backend.
+
+----
+$ source /path/to/examplerc
+$ mkdir ~/images
+$ sudo cp /path/to/downloaded/rhel-server-7.5-x86_64-kvm.qcow2 ~/images
+$ cd ~/images
+$ qemu-img convert \
+    -f qcow2 -O raw \
+    rhel-server-7.5-x86_64-kvm.qcow2 ./<img-name>.raw
+$ openstack image create <img-name> \
+    --disk-format=raw \
+    --container-format=bare < <img-name>.raw
+----
+
+A {rhosp} administrator may allow this `glance` {rhel} image to be readily
+available to all projects within the {rhosp} environment. This may be done by
+`source` of an RC file with OpenStack administrator privileges that include the
+ `--public` option prior to creating the `raw` image.
+
+Confirm the creation of the image via `openstack image list`. An example output
+is shown.
+
+----
+$ openstack image list
++--------------------------------------+-------+--------+
+| ID                                   | Name  | Status |
++--------------------------------------+-------+--------+
+| 7ce292d7-0e7f-495d-b2a6-aea6c7455e8c | rhel7 | active |
++--------------------------------------+-------+--------+
+----
```

Comment:
finish to verify this section

Reply:
Not sure what you want here as an example is shown.